client/components/payment-logo/docs/example.jsx

```diff
@@ -17,9 +17,12 @@ class PaymentLogoExamples extends React.PureComponent {
 	render() {
 		return (
 			<div>
-				<PaymentLogo type="amex" /> <PaymentLogo type="discover" /> {' '}
-				<PaymentLogo type="mastercard" /> <PaymentLogo type="visa" /> {' '}
-				<PaymentLogo type="paypal" isCompact /> <PaymentLogo type="paypal" />
+				<PaymentLogo type="alipay" /> <PaymentLogo type="amex" /> <PaymentLogo type="bancontact" />{' '}
+				<PaymentLogo type="diners" /> <PaymentLogo type="discover" /> <PaymentLogo type="eps" />{' '}
+				<PaymentLogo type="giropay" /> <PaymentLogo type="ideal" /> <PaymentLogo type="jcb" />{' '}
+				<PaymentLogo type="mastercard" /> <PaymentLogo type="p24" /> <PaymentLogo type="paypal" />
+				<PaymentLogo type="paypal" isCompact /> <PaymentLogo type="unionpay" />{' '}
+				<PaymentLogo type="visa" />
```

Comment:
Thank you @jsnmoon . I've redone this part with array maps as you suggested. I've also moved out the special case: `type="placeholder"` to another paragraph.

For right margins, my intuition was to use add a CSS class for this `payment-logo/docs/example.jsx` that append margins to each `<PaymentLogo />`. However, I failed to find any example that introduces any new CSS classes just for documentation purpose. I guess that it's not encouraged?

To workaround, I wrapped each `<PaymentLogo />`  with a `<div>` so that they are displayed in different lines.

Reply:
> For right margins, my intuition was to use add a CSS class for this `payment-logo/docs/example.jsx` that append margins to each `<PaymentLogo />`. However, I failed to find any example that introduces new CSS classes just for documentation purpose. I guess that it's not encouraged?

Unfortunately, I don't think we currently have a good way of supporting this yet.

> To workaround, I wrapped each `<PaymentLogo />`  with a `<div>` so that they are displayed in different lines.

Well done! 👍